static/js/components/PersonalTab.js

```diff
@@ -68,7 +68,15 @@ export default class PersonalTab extends React.Component {
     dispatch(setProfileStep(PERSONAL_STEP));
   }
 
-  onChange = (selection: Option): void => {
+  componentDidUpdate() {
+    const { currentProgramEnrollment } = this.props;
+    const selectedProgram = this.getSelectedProgramId();
+    if ( currentProgramEnrollment && !selectedProgram ) {
+      this.onProgramSelect({ value: currentProgramEnrollment.id, label: '' });
```

Comment:
The onProgramSelect is passed in as an onChange handler but it doesn't receive an event as an argument, just an Option object. I wanted to reuse the code that's in onProgramSelect here, but if you think we shouldn't re-use that function I'm fine with changing it.